src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -0,0 +1,35 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Analyzers.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers )' == '' And 
+                                   '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And
+                                   $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0'))">true</EnableNETAnalyzers>
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers )' == ''">false</EnableNETAnalyzers>
+  </PropertyGroup>
+  <Import Project="$([System.IO.Path]::GetFullPath($(MSBuildThisFileDirectory)..))\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.targets"
+          Condition="$(EnableNETAnalyzers)" />
+  <ItemGroup>
```

Comment:
I would put an `EnableNETAnalyzers` condition on the `ItemGroup` so you don't have to repeat that clause for each item.

Reply:
good idea